admin-client/src/pages/domain/Show.svelte

```diff
@@ -32,7 +32,10 @@
   }
 
   function deprovision() {
-    domainPromise = deprovisionDomain(id);
+    // eslint-disable-next-line no-alert
+    if (window.confirm('Are you sure you want to destroy this domain?')) {
```

Comment:
```suggestion
    if (window.confirm('Are you sure you want to deprovision this domain?')) {
```

Reply:
Oh for heaven's sake. I was Ctrl-Z-ing back and forth on my way to this as I resolved a different lint thing and I obviously missed one Cmd-Ctrl-Z and lost this. Thanks for catching it, Dave.